src/main/scala/org/youshuren/config/AppConfig.scala

```diff
@@ -0,0 +1,22 @@
+package org.youshuren.config
+
+import com.typesafe.config.ConfigFactory
+
+object AppConfig {
```

Comment:
You should be interested in the config-loading [library](https://github.com/pureconfig/pureconfig)

Reply:
looks nice, will have a try